cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,11 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-    
+    cy.visit('https://react-redux.realworld.io');
+    cy.get(':nth-child(2) > .nav-link').click();
+    cy.get(':nth-child(1) > .form-control').type('test1412@gmail.com');
+    cy.get(':nth-child(2) > .form-control').type('test1412');
+    cy.get('.btn').click();
+    cy.get(':nth-child(4) > .nav-link')
```

Comment:
Please assert that there is username in this element as it is required in the task description. use method `should` for it